playwright/_impl/_transport.py

```diff
@@ -34,12 +34,15 @@
 
 # Sourced from: https://github.com/pytest-dev/pytest/blob/da01ee0a4bb0af780167ecd228ab3ad249511302/src/_pytest/faulthandler.py#L69-L77
 def _get_stderr_fileno() -> Optional[int]:
-    if sys.stderr.closed:
-        return None
-
     try:
+        if sys.stderr.closed:
+            return None
+
         return sys.stderr.fileno()
     except (AttributeError, io.UnsupportedOperation):
+        # pyinstaller apps may not have stderr
```

Comment:
Isn't it pythonw specific?

Reply:
Yes, you're right.  I was focused only on pyinstaller but it is specific to pythonw.
I will update the comment.  Would you prefer this scenario to be handled differently in the method?

Reply:
It seems alright, but we could also check above if it's a nullwriter which seems cleaner.

Reply:
I've updated the branch as `pythonw` and `pyinstaller` scenarios are different.

- In case of `pythonw`, `sys.stderr ` is `None`
- In case of `pyinstaller`, `sys.stderr` is an instance of `NullWriter` class from the `pyinstaller` package.

Moved the check for these two situations to early in the `try` clause.  Don't explicitly check for `NullWriter` class as this is from another package so added a generic check if `sys.stderr` has attribute `closed`.